src/index.html

```diff
@@ -6,7 +6,55 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="container">
+    <div class="calendar container__calendar">
+      <div class="calendar__days">
+        <div class="calendar__day calendar__day--1"></div>
```

Comment:
```suggestion
        <div class="calendar__day"></div>
```
Remove it everywhere. Use nth-child instead

Reply:
fixed